service/src/services/domain/builders/community-platform-invitation-created/community.platform.invitation.created.notification.builder.ts

```diff
@@ -9,6 +9,9 @@ import { EmailTemplate } from '@common/enums/email.template';
 import { CommunityPlatformInvitationCreatedEmailPayload } from '@common/email-template-payload';
 import { AlkemioUrlGenerator } from '@src/services/application/alkemio-url-generator/alkemio.url.generator';
 
+// defined in client-web
+const INVITATIONS_PATH = '?dialog=invitations';
```

Comment:
extract this to the config - `notifications.yml`. That way we do not need to deploy it if the path changes